content/hi/docs/contribute/new-content/new-features.md

```diff
@@ -0,0 +1,81 @@
+---
+title:  एक रिलीज के लिए एक सुविधा का प्रलेखीकरण
+linktitle: रिलीज़ के लिए प्रलेखीकरण
+content_type: concept
+main_menu: true
+weight: 20
+card:
+  name: contribute
+  weight: 45
+  title:  Documenting a feature for a release
+---
+<!-- overview -->
+
+प्रत्येक प्रमुख कुबेरनेट्स रिलीज़ नई सुविधाएँ शामिल होती हैं जिनके लिए प्रलेखन की आवश्यकता होती है। नई रिलीज़ मौजूदा सुविधाओं और प्रलेखन में भी अपडेट लाती हैं (जैसे किसी सुविधा को अल्फा से बीटा में अपग्रेड करना)।
+
+आम तौर पर, सुविधा के लिए जिम्मेदार SIG सुविधा के ड्राफ्ट प्रलेखन को `kubernetes/website` रिपॉजिटरी की उपयुक्त विकास शाखा को पुल अनुरोध के रूप में प्रस्तुत करता हैं, और SIG डॉक्स टीम में कोई व्यक्ति संपादकीय प्रतिक्रिया प्रदान करता हैं, या ड्राफ्ट को सीधे संपादित करता हैं। इस खंड में दोनों समूहों द्वारा रिलीज के दौरान उपयोग की जाने वाली ब्रांचिंग परम्पराएं और प्रक्रिया शामिल हैं।
+
+
+
+<!-- body -->
+
+## प्रलेखीकरण योगदानकर्ताओं के लिए
+
+सामान्य तौर पर, प्रलेखीकरण योगदानकर्ता किसी रिलीज़ के लिए शुरू से ही सामग्री नहीं लिखते हैं।
+इसके बजाय, वे मसौदा प्रलेखन को परिष्कृत करने और इसे जारी करने के लिए, तैयार करने के लिए, एक नई सुविधा बनाने वाले SIG के साथ काम करते हैं।
+
+प्रलेखन या सहायता के लिए एक सुविधा चुनने के बाद, इसके बारे में `#sig-docs` स्लैक चैनल में, साप्ताहिक SIG डॉक्स मीटिंग में, या सीधे सुविधा SIG द्वारा दायर PR पर पूछें। यदि आपको आगे बढ़ने की अनुमति दी गई हैं, तो आप [किसी अन्य व्यक्ति के PR में प्रतिबद्ध](/docs/contribute/review/for-approvers/#commit-into-another-persons-pr) में वर्णित तकनीकों में से एक का उपयोग करके PR में संपादित कर सकते हैं।
```

Comment:
```suggestion
प्रलेखन या सहायता के लिए एक सुविधा चुनने के बाद, इसके बारे में `#sig-docs` स्लैक चैनल में, साप्ताहिक SIG डॉक्स मीटिंग में, या सीधे SIG सुविधा द्वारा PR पर पूछें। यदि आपको आगे बढ़ने की अनुमति दी गई हैं, तो आप वर्णित तकनीकों में से एक का उपयोग करके [किसी अन्य व्यक्ति के PR में कमिट](/docs/contribute/review/for-approvers/#commit-into-another-persons-pr) करके PR में संपादित कर सकते हैं।
```

Reply:
That link also needs to be updated, as this link is redirecting towards english page. But currently as that page in hindi doesn't exists. So, its fine for this PR to mention english docs link